Documentation/BuildInstructions.md

```diff
@@ -106,6 +106,23 @@ Then use this script: `nix-shell myshell.nix`.
 
 Once you're in nix-shell, you should be able to follow the build directions.
 
+#### Alpine Linux
+First, make sure you have enabled the `community` repository in `/etc/apk/repositories` and run `apk update`. It has been tested on `edge`, YMMV on `stable`.
+
+```console
+# the basics, if you have not already done so
+apk add bash curl git sudo
+
+# rough equivalent of build-essential
+apk add build-base
+
+# qemu
+apk add qemu qemu-system-i386 qemu-img qemu-ui-gtk
+
+# build tools (samurai is a drop-in replacement for ninja)
+apk add cmake e2fsprogs samurai mpc1-dev mpfr-dev gmp-dev
```

Comment:
`util-linux` is required to build GRUB image (in case it works, that is), so should this be added too?

Reply:
Hmm. Even with `util-linux` it fails.

```
installing grub using /usr/sbin/grub-install with /home/jagger/Projects/serenity/Meta/grub-mbr.cfg...
/usr/sbin/grub-install: error: /usr/lib/grub/i386-pc/modinfo.sh doesn't exist. Please specify --target or --directory.
```

Reply:
Ah, `grub-bios` does the trick. My Alpine installation is on EFI so I didn't have the package.

Reply:
Yup, and also `parted`, but script won't work with busybox's `losetup`, which can be confusing, so that's why I'm suggesting adding `util-linux`